src/main/java/org/wildfly/security/auth/server/IdentityLocator.java

```diff
@@ -31,7 +33,7 @@
  *
  * @author <a href="mailto:david.lloyd@redhat.com">David M. Lloyd</a>
  */
-public final class IdentityLocator {
+public final class IdentityLocator implements Serializable {
```

Comment:
I think making this implement Serializable could be problematic - maybe lets scale this back to single node for now and then work with the clustering engineers on what this means in a distributed environment.